phpBB/ext/phpbb/teamsecurity/acp/teamsecurity_module.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ *
+ * phpBB Team Security Measures
+ *
+ * @copyright (c) 2014 phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\teamsecurity\acp;
+
+class teamsecurity_module
+{
+	/** @var \phpbb\config\config */
+	protected $config;
+
+	/** @var \phpbb\db\driver\driver_interface */
+	protected $db;
+
+	/** @var \phpbb\log\log */
+	protected $log;
+
+	/** @var \phpbb\request\request */
+	protected $request;
+
+	/** @var \phpbb\template\template */
+	protected $template;
+
+	/** @var \phpbb\user */
+	protected $user;
+
+	/** @var string */
+	public $u_action;
+
+	public function __construct()
+	{
+		global $config, $db, $phpbb_log, $request, $template, $user;
+
+		$this->config = $config;
+		$this->db = $db;
+		$this->log = $phpbb_log;
+		$this->request = $request;
+		$this->template = $template;
+		$this->user = $user;
+
+		$this->user->add_lang('acp/common');
+		$this->user->add_lang_ext('phpbb/teamsecurity', 'acp_teamsecurity');
+	}
+
+	/**
+	 * Main ACP module
+	 *
+	 * @param int $id
+	 * @param string $mode
+	 * @return null
+	 * @access public
+	 */
+	public function main($id, $mode)
+	{
+		$this->tpl_name = 'acp_teamsecurity';
+		$this->page_title = $this->user->lang('ACP_TEAM_SECURITY_SETTINGS');
+
+		$form_key = 'acp_teamsecurity';
+		add_form_key($form_key);
+
+		if ($this->request->is_set_post('submit'))
+		{
+			if (!check_form_key($form_key))
+			{
+				trigger_error($this->user->lang('FORM_INVALID') . adm_back_link($this->u_action), E_USER_WARNING);
+			}
+
+			$this->config->set('sec_login_email', $this->request->variable('sec_login_email', 0));
+			$this->config->set('sec_login_attempts', $this->request->variable('sec_login_attempts', 0));
+			$this->config->set('sec_strong_pass', $this->request->variable('sec_strong_pass', 0));
+			$this->config->set('sec_min_pass_chars', $this->request->variable('sec_min_pass_chars', 0));
+			$this->config->set('sec_usergroups', serialize($this->request->variable('sec_usergroups', array(0))));
```

Comment:
This is part of the extension. You're welcome to comment on it separately but doesn't really affect this PR unless it's a security issue.